PayKids/presentation/src/main/java/com/paykids/presentation/view/quiz/QuizEntryFragment.kt

```
@@ -0,0 +1,12 @@
+package com.paykids.presentation.view.quiz
+
+import com.paykids.presentation.base.BaseFragment
+import com.paykids.presentation.databinding.FragmentMypageBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class QuizEntryFragment: BaseFragment<FragmentMypageBinding>() {
+    override fun initView() {
+
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)